DSA/Graph/Graph.cpp

```diff
@@ -0,0 +1,97 @@
+#include <iostream>
+#include <vector>
+using namespace std;
+
+class Graph {
+private:
+    int num_vertices;
+    vector<vector<int>> adj_list;
```

Comment:
Avoid use inbuilt libraries or STL / Collections. Use adjacency matrix or Linked list for adjacency list.

Reply:
resolved